### ISSUE TITLE
Cannot install RuboCop 0.58.{1,2}

### ISSUE THREAD

**Issue by boris-petrov** [jruby/jruby#5260]
JRuby version 9.2.0.0.

In [this](https://github.com/rubocop-hq/rubocop/commit/e41a2d5bd5b33cba1219e0f798397fac9b8fe543) commit, RuboCop added a filter on the `parser` dependency version. Then they released 0.58.1. Now they've released 0.58.2 and I cannot update to either version. The error that I'm getting is:

```
> Could not resolve rubygems:parser:(=2.5.1.1,).
  Required by:
      project :features > rubygems:rubocop:0.58.2
   > there is no overlap for (=2.5.1.1,99999) and (=2.5.1.1,)
```

I'm not sure whether this is a JRuby bug or a Gradle one or what is going on but I think I won't be able to upgrade RuboCop any more at all until this is fixed. :)

Any ideas?

**Comment by mkristian** [comment]
this is gradle bug, version range parsing seems a bit odd here.

**Comment by boris-petrov** [comment]
You suggest I open a bug with the Gradle guys?

**Comment by mkristian** [comment]
yes, with https://github.com/jruby-gradle/jruby-gradle-plugin I will pick it from there. it is just not a jruby problem what you have.

**Comment by koic** [comment]
There was a bug in Parser 2.5.1.1, so RuboCop does not use that version.
https://github.com/rubocop-hq/rubocop/issues/6092

```ruby
s.add_runtime_dependency('parser', '>= 2.5', '!= 2.5.1.1')
```

https://github.com/rubocop-hq/rubocop/blob/v0.58.2/rubocop.gemspec#L39

Please run `bundle update` and update Parser gem.
